test/test-h5.jl

```diff
@@ -75,9 +75,14 @@ h5path = joinpath(resdir, "hdf5-files", "20220914T1244.aqua.labeled_image.250m.h
     @test typeof(props) == Matrix{Float64}
     close(fid)
 
+    @test_throws "can't be represented" choose_dtype(-1) 
     @test choose_dtype(100) == UInt8
     @test choose_dtype(300) == UInt16
     @test choose_dtype(70_000) == UInt32
+    @test choose_dtype(18_446_744_073_709_551_615) == UInt64
+    @test choose_dtype(18_446_744_073_709_551_616) == UInt128
```

Comment:
Can 2^64-1 and 2^64 be used here?
```suggestion
    @test choose_dtype(BigInt(2)^64 - 1) == UInt64
    @test choose_dtype(BigInt(2)^64) == UInt128
```